### PR TITLE
Fix up KaliteSetupScript.iss

### PR DESCRIPTION
**Pull request by MCGallaspy**
An issue with the encoding, presumably...?

### CODE REVIEW

File: windows/installer-source/KaliteSetupScript.iss [hunk 1]
```diff
@@ -342,7 +342,7 @@ var
 
 begin
     PipPath := 'C:\Python27\Scripts\pip.exe';
-    ﻿PipCommand := 'install ' + '"' + ExpandConstant('{app}\ka-lite\dist\ka-lite-static-')  + '{#MyVersion}' + '.zip' + '"';
+    PipCommand := 'install ' + '"' + ExpandConstant('{app}\ka-lite\dist\ka-lite-static-')  + '{#MyVersion}' + '.zip' + '"';
```
Comment (MCGallaspy) [inline windows/installer-source/KaliteSetupScript.iss]: Also, `{#MyVersion}` returns a different version string than the output of `setup.py sdist --static` (namely `0.14alpha1` vs `0.14a1`). Let me try to resolve that in this PR as well.

**Comment by MCGallaspy** [comment]
Fixes #69.

**Comment by MCGallaspy** [comment]
Fixes #71 also.

**Comment by aronasorman** [comment]
Have you tested the changes in here?

**Comment by MCGallaspy** [comment]
I've tested everything except the latest commit. It's exhibiting different behavior from earlier 0.14.x builds -- previously the db setup failed almost instantaneously, and now it's taking a loooooong time. But that's consistent with versions, so I'm taking it as a good sign.

**Comment by MCGallaspy** [comment]
Yeah, I think that step is hanging because it's waiting for input or something silly. Will amend again.

**Comment by MCGallaspy** [comment]
Tested and it works.

**Pull request merged**